public/favicon.svg

```diff
@@ -1,13 +1,36 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
-    <defs>
-        <linearGradient id="grad" x1="0%" y1="0%" x2="100%" y2="100%">
-            <stop offset="0%" style="stop-color:#7c3aed"/>
-            <stop offset="100%" style="stop-color:#a855f7"/>
-        </linearGradient>
-    </defs>
-    <rect width="32" height="32" fill="url(#grad)" rx="4"/>
-    <rect x="4" y="24" width="4" height="8" fill="#c084fc" rx="0.96"/>
-    <rect x="10" y="20" width="4" height="12" fill="#c084fc" rx="0.96"/>
-    <rect x="16" y="16" width="4" height="16" fill="#c084fc" rx="0.96"/>
-    <rect x="22" y="12" width="4" height="20" fill="#c084fc" rx="0.96"/>
-</svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
the icon is white, so in the light mode it disappears, add an alt black version and choose depending on the theme
<img width="459" height="87" alt="image" src="https://github.com/user-attachments/assets/5ce70826-1864-462b-a912-2b9b95e94b9c" />